internal/service/create/options.go

```diff
@@ -0,0 +1,57 @@
+package create
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	commonerrors "github.com/kyma-project/modulectl/internal/common/errors"
+	iotools "github.com/kyma-project/modulectl/tools/io"
+)
+
+type Options struct {
+	Out                  iotools.Out
+	ModuleConfigFile     string
+	Credentials          string
+	GitRemote            string
+	Insecure             bool
+	TemplateOutput       string
+	RegistryURL          string
+	RegistryCredSelector string
+	SecScannerConfig     string
+}
+
+func (opts Options) validate() error {
```

Comment:
If you introduce this validation func already, please also provide the unit tests for it in internal/service/create/options_test.go within the `create_test` package. Thanks! ✌️